apps/src/code-studio/pd/international_opt_in/InternationalOptIn.jsx

```diff
@@ -72,7 +72,11 @@ class InternationalOptInComponent extends FormComponent {
    * @returns {boolean}
    */
   isColombiaSelected() {
-    return this.props.data && this.props.data.schoolCountry === 'colombia';
+    return (
+      this.props.data &&
+      this.props.data.schoolCountry &&
```

Comment:
Can we use [optional chaining](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Optional_chaining) (i.e. `this.props.data.schoolCountry?.toLowerCase()`) now that we're dropping support for IE?

Reply:
ooh, thanks for this tip! I've changed both Colombia and Chile checks